src/System.Reflection.Metadata/tests/Metadata/Ecma335/Encoding/BlobEncodersTests.cs

```diff
@@ -388,7 +388,350 @@ public void FixedArgumentsEncoder_AddArgument()
         // NamedArgumentTypeEncoder
         // CustomAttributeArrayTypeEncoder
         // CustomAttributeElementTypeEncoder
-        // SignatureTypeEncoder
+
+        [Fact]
+        public void SignatureTypeEncoder_Primitives()
+        {
+            var b = new BlobBuilder();
+            var e = new SignatureTypeEncoder(b);
+            Assert.Same(b, e.Builder);
+
+            e.Boolean();
+            AssertEx.Equal(new byte[] { 0x02 }, b.ToArray());
```

Comment:
Does it display sequence diff if the assert fails? I find some I the default xunit asserts annoying, e.g. Equals on a string trims the actual and expected strings in the displayed error.